.flake8

```diff
@@ -0,0 +1,8 @@
+[flake8]
+exclude = docs/*
+max-line-length = 88
+extend-ignore = E203
+ignore = E501,W504,W503
+per-file-ignores =
+    **/__init__.py:F401
+    **/tests/*:D
```

Comment:
what is the D class of errors?

Reply:
Removed:
https://github.com/alteryx/nlp_primitives/pull/201/commits/d5a5f3114685cdf36b79e258b4241c6c8b5486d2